api/pom.xml

```diff
@@ -6,7 +6,7 @@
     <parent>
         <groupId>org.jboss.arquillian.extension</groupId>
         <artifactId>arquillian-performance-parent</artifactId>
-        <version>1.0.0.CR2-SNAPSHOT</version>
```

Comment:
Please don't update library version. We'll do that on release. Same goes for all arquillian-extension-performance modules.